js/templates/pageNav.html

```diff
@@ -8,7 +8,7 @@ <h3 class="margin0 marginTop25"><%= polyglot.t('Discover') %></h3>
   </a>
 </div>
 
-<div class="popMenu hide popMenu-navBar js-navProfileMenu">
+<div class="popMenu popMenu-navBar js-navProfileMenu">
```

Comment:
You could probably get away with setting min-height to zero, and then changing it to a large number. That would give you a similar unfolding effect. 

Or we just set a fixed height for the list container, and always use that regardless of how many messages there are.